products/nextjs.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Next.js
+category: framework
+tags: javascript-runtime
+iconSlug: nextdotjs
+permalink: /nextjs
+alternate_urls:
+-   /next-js
+-   /nextdotjs
+versionCommand: npx next --version
+releasePolicyLink: https://github.com/vercel/next.js/blob/canary/contributing/repository/release-channels-publishing.md
+changelogTemplate: https://github.com/vercel/next.js/releases/latest
+activeSupportColumn: true
```

Comment:
```suggestion
```

It does not seem Next.js is following a two-phases support  (active support then security support).  All `support` fields on releases can be removed too.